src/components/Notification/Notification.tsx

```diff
@@ -376,6 +376,8 @@ const NotificationDropdown: React.FC = () => {
     useState(false)
   const [notification, setNotification] = useState<Notification>()
 
+  const dropdownRef = useRef<HTMLDivElement>(null)
+
```

Comment:
```suggestion
  const buttonRef = useRef<HTMLButtonElement>(null)
```
현재 알림 클릭시 후, 알림 아이콘을 다시 클릭하면 닫혀야 하는데, 외부 클릭과 알림클릭시 이벤트가 동시에 작동해 닫히고 다시 열리는 오류가 존재합니다. 그 해결을 위한 ref추가 입니다.